releases/v19.2.0-alpha.20190701.md

```diff
@@ -0,0 +1,155 @@
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-alpha.20190701.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-alpha.20190701.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-alpha.20190701.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-alpha.20190701.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-alpha.20190701
+~~~
+
+### Backward-incompatible changes
+
+- Integer and float columns of less than the max width will now be returned as their own type via the binary protocol. For example, an `int4` column will be returned in 32 bits over the pgwire binary protocol instead of 64 bits. [#37295][#37295]
+
+### General changes
+
+- Remove `kv.range_merge.manual_split_ttl` [cluster setting](../v19.2/cluster-settings.html). [#38004][#38004]
+- Upgrade the build environment to use go1.12.5. [#37966][#37966]
+
+### Enterprise edition changes
+
+- Fixes help text that erroneously labeled [Encryption at Rest](../v19.2/encryption.html) as experimental. [#38228][#38228]
+
+### SQL language changes
+
+- An initial prototype of support for using `IMPORT INTO` to ingest CSV data into an existing table. Usage: `IMPORT INTO t(a, b) CSV DATA ('gs://acme-co/employees.csv')`. [#37451][#37451]
+- Implements sql support for timestamp objects to have an optional precision value. [#37920][#37920]
+- The logging format for statement execution (as enabled by `sql.trace.log_statement_execute`) and audit logs (as enabled by [`ALTER TABLE .. EXPERIMENTAL AUDIT SET ...`](../v19.2/experimental-audit.html)) is extended with one column that reports internal execution retries performed automatically by the server. The value zero is logged for the first retry attempt and thus corresponds to statements directly received from a client. [#38028][#38028]
+- Add `WITH EXPIRATION` option for [`SPLIT AT`](../v19.2/split-at.html). [#38004][#38004]
+- The `DISTINCT` keyword is now rejected when used in a window function. It was previously accepted, but ignored. [#37912][#37912]
+- Add `UNSPLIT ALL` statement. [#38023][#38023]
+- The `relam` column of the `pg_catalog` tables `pg_class` and `pg_index` is now filled in correctly. [#38164][#38164]
```

Comment:
Just checking: should `relam` be `realm`?

Reply:
Great question! I double checked and it's a thing: https://www.postgresql.org/docs/9.6/catalog-pg-class.html